app/models/contact.rb

```diff
@@ -0,0 +1,11 @@
+class Contact < ApplicationRecord
+    validates :name, presence: true, length: { minimum: 2 }
```

Comment:
Perhaps we enforce some max limits for these fields (100-255 for smaller fields, 10,000 for message)?